js/src/components/google-combo-account-card/account-creation-description/index.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useGoogleAccount from '.~/hooks/useGoogleAccount';
+import useGoogleMCAccount from '.~/hooks/useGoogleMCAccount';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+
+/**
+ * Renders the description for the account creation card.
+ *
+ * @param {Object} props Props.
+ * @param {boolean} props.accountsCreated Whether accounts have been created.
+ * @param {boolean} props.isCreatingBothAccounts Whether both, MC and Ads accounts are being created.
+ * @param {boolean} props.isCreatingOnlyMCAccount Whether only the Merchant Center account is being created.
+ * @param {boolean} props.isCreatingOnlyAdsAccount Whether only Google Ads account is being created.
+ */
+const AccountCreationDescription = ( {
+	accountsCreated,
```

Comment:
@ankitguptaindia We won't need this prop if we only display the MC and Ads IDs when they are connected. See my comment further below.